container/root/etc/cont-finish.d/00-nginx.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/execlineb -P
+
+foreground { nginx -s quit }
```

Comment:
@bryanlatten : Does it make sense to trap the exit code here somehow to see that it actually ended gracefully or do we feel that if the process `nginx -s quit` returns with a non-zero exit code, it'll just abort at that point and return? If so, then if that log message is missing the assumption is that it didn't die gracefully. I guess it'll also print something like

```
[cont-finish.d] executing container finish scripts...
[cont-finish.d] failed or something.
```

Does that sound right?

Reply:
@adobejmong the default behavior of foreground is that the exit code is returned as the exit code of the script: https://www.skarnet.org/software/execline/foreground.html

you would see the message: 
`[cont-finish.d] script 00-nginx.sh exit code 1`